### PR TITLE
Pass millisecond duration correctly to Broadcaster

### PR DESCRIPTION
**Pull request by thomshutt**

### CODE REVIEW

File: transcode/manifest.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func GetSourceSegmentURLs(sourceManifestURL string, manifest m3u8.MediaPlaylist)
 			urls,
 			SourceSegment{
 				URL:            u,
-				DurationMillis: int64(segment.Duration),
+				DurationMillis: int64(segment.Duration * 1000),
```
Comment (thomshutt) [inline transcode/manifest.go:67]: This is where the issue was - the duration in the source manifest is in Seconds

File: transcode/transcode.go [hunk 2]
```diff
@@ -231,7 +231,7 @@ func transcodeSegment(segment segmentInfo, streamName, manifestID string, transc
 	}
 
 	for _, stats := range transcodedStats {
-		stats.BitsPerSecond = uint32(float64(stats.Bytes) * 8000.0 / float64(stats.DurationMs))
+		stats.BitsPerSecond = uint32(float64(stats.Bytes) * 8000.0 / float64(stats.DurationMs/1000))
```
Comment (thomshutt) [inline transcode/transcode.go:234]: Had to update our bitrate calculation, which is bits / second but uses a millisecond variable to calculate it

File: transcode/manifest.go (hunk 1 above)
Comment (yondonfu) [reply transcode/manifest.go:67]:
```suggestion
				DurationMillis: int64(segment.Duration * time.Second),
```

WDYT?

File: transcode/transcode.go (hunk 2 above)
Comment (yondonfu) [reply transcode/transcode.go:234]:
```suggestion
		stats.BitsPerSecond = uint32(float64(stats.Bytes) * 8000.0 / float64(stats.DurationMs/time.Second))
```

WDYT?

**Review by yondonfu — APPROVED** [review]
Just 2 nits - feel free to choose to address before merge.

LGMT

File: transcode/manifest.go (hunk 1 above)
Comment (thomshutt) [reply transcode/manifest.go:67]: Much nicer!

**Review by emranemran — APPROVED** [review]

**Pull request merged**